analyzers/tests/SonarAnalyzer.UnitTest/Helpers/DiagnosticDescriptorBuilderTest.cs

```diff
@@ -18,86 +18,13 @@
 {
     [TestClass]
     public class DiagnosticDescriptorBuilderTest
     {
-        private const string LanguageValue = "language";
-
-        [TestMethod]
-        public void GetHelpLink_CSharp()
-        {
-            var helpLink = DiagnosticDescriptorBuilder
-                .GetHelpLink(csharp.SonarAnalyzer.RspecStrings.ResourceManager, "S1234");
-            helpLink.Should().Be("https://rules.sonarsource.com/csharp/RSPEC-1234");
-        }
-
-        [TestMethod]
-        public void GetHelpLink_VisualBasic()
-        {
-            var helpLink = DiagnosticDescriptorBuilder
-                .GetHelpLink(vbnet.SonarAnalyzer.RspecStrings.ResourceManager, "S1234");
-            helpLink.Should().Be("https://rules.sonarsource.com/vbnet/RSPEC-1234");
-        }
-
-        [TestMethod]
-        public void GetDescriptor_SetsIsEnabledByDefaultToGivenValue()
-        {
-            // Arrange
-            var diagnosticId = "FooBar";
-            var mockedResourceManager = CreateMockedResourceManager(diagnosticId, isActivatedByDefault: true);
-
-            DiagnosticDescriptorBuilder.GetDescriptor(diagnosticId, "", mockedResourceManager, true)
-                .IsEnabledByDefault
-                .Should()
-                .BeTrue();
-
-            DiagnosticDescriptorBuilder.GetDescriptor(diagnosticId, "", mockedResourceManager, false)
-                .IsEnabledByDefault
-                .Should()
-                .BeFalse();
-
-            DiagnosticDescriptorBuilder.GetDescriptor(diagnosticId, "", mockedResourceManager)
-                .IsEnabledByDefault
-                .Should()
-                .BeTrue();
-        }
-
-        [TestMethod]
-        public void GetDescriptor_WhenIsActivatedByDefaultAndIdeVisibilityNotHidden_HasOnlySonarWayAndLanguageTags()
-        {
-            // Arrange
-            var diagnosticId = "foo";
-            var mockedResourceManager = CreateMockedResourceManager(diagnosticId, true);
-
-            // Act
-            var result = DiagnosticDescriptorBuilder.GetDescriptor(diagnosticId, "", mockedResourceManager);
-
-            // Assert
-            result.CustomTags.Should().OnlyContain(DiagnosticDescriptorBuilder.SonarWayTag, DiagnosticDescriptorBuilder.MainSourceScopeTag, LanguageValue);
-        }
-
-        [TestMethod]
-        public void GetDescriptor_WhenIsNotActivatedByDefaultAndIdeVisibilityNotHidden_ContainsOnlyLanguage()
-        {
-            // Arrange
-            var diagnosticId = "foo";
-            var mockedResourceManager = CreateMockedResourceManager(diagnosticId, false);
-
-            // Act
-            var result = DiagnosticDescriptorBuilder.GetDescriptor(diagnosticId, "", mockedResourceManager);
-
-            // Assert
-            result.CustomTags.Should().OnlyContain(DiagnosticDescriptorBuilder.MainSourceScopeTag, LanguageValue);
-        }
-
         [TestMethod]
         public void GetUtilityDescriptor_Should_Contain_NotConfigurable_CustomTag()
```

Comment:
Nitpick: use `string.Empty` for empty string.

Reply:
Done in #5858